Liberland-constitution.md

```diff
@@ -71,13 +71,13 @@ The Free Republic of Liberalnd shall be governed by the Public Authority in acco
 ### Article IV: The Legislative Power
 
 * **§IV.1.** All legislative powers of the Free Republic of Liberland shall be vested in the Citizens of the Free Republic of Liberland and the Assembly of the Free Republic of Liberland, hereinafter, "the Assembly", which shall exercise it on behalf of the Citizens.
-* **§IV.2.** The Assembly shall consist of twenty Assembly Representatives elected by the Citizens in the General Election held every four years.
+* **§IV.2.** The Assembly shall consist of at least twenty Assembly Representatives elected by the Citizens in the General Election held every four years. To ensure the Citizens' right to fair representation, there shall be one representative for every ten thousand citizens.
 * **§IV.3.** No Assembly Representative shall hold the office for more than eight years.
 * **§IV.4.** The Assembly shall elect the Chancellor of the Free Republic of Liberland, hereinafter, “the Chancellor”, who shall form the Cabinet of the Free Republic of Liberland, hereinafter, "the Cabinet", with the majority of no fewer than eleven votes; all members of the Cabinet shall be drawn from citizenry.
 * **§IV.5.** The Assembly shall have the power to withdraw confidence from the Chancellor along with his Cabinet by passing the constructive motion of no confidence with the majority of no fewer than eleven votes, only if there is a majority for a prospective successor; confidence from a single Member of the Cabinet can be withdrawn by passing the ordinary motion of no confidence with the majority of no fewer than eleven votes.
 * **§IV.6.** If no Chancellor is elected in accordance with the procedure provided in §IV.4 for more than ninety days, the General Election must be held.
 * **§IV.7.** The power to initiate a legislative procedure shall lie with the Cabinet only.
-* **§IV.8.** An ordinary Bill shall be passed into law with the majority of no fewer than fifteen votes of the Assembly Representatives subject to the power of general veto by the Citizens as provided in §IV.13 and §IV.14.
+* **§IV.8.** An ordinary Bill shall be passed into law with the majority of no fewer than three-quarters of the votes of the Assembly Representatives subject to the power of general veto by the Citizens as provided in §IV.13 and §IV.14.
 * **§IV.9.** A Constitutional Bill proposing an Amendment to this Constitution shall be passed only with the unanimous vote with all Assembly Representatives present and participating in the vote, and shall be subject to the power of general veto by the Citizens as provided in §IV.15.
```

Comment:
I'm not saying if this is a good or bad thing, in fact it might be an awesome positive effect of this change, but it would become next to impossible to achieve a unanimous vote on just about anything once the Assembly grows to a few hundred members. This in turn would make a constitutional amendment next to impossible.

This could be very good, because it would make the legal erosion of the freedoms spelled out in the Liberland Constitution next to impossible.

But it would also make any amendment to the Constitution effectively impossible.